tests/Feature/Controllers/TasksControllerTest.php

```
@@ -0,0 +1,218 @@
+
+        /** Assert */
+        $response->assertOk();
+        $response->assertViewIs('projects::tasks_index');
+        $response->assertViewHas('tasks');
+        $response->assertViewHas('task_statuses');
+
+        /** Verify task is in the list */
+        $tasks = $response->viewData('tasks');
+        $taskIds = $tasks->pluck('task_id')->toArray();
+        $this->assertContains($task->task_id, $taskIds);
+    }
```

PHPMD (2.15.0)

The method it_displays_list_of_tasks is not named in camelCase. (undefined)

(CamelCaseMethodName)